cmrf_views/src/Plugin/views/query/API4.php

```diff
@@ -215,7 +230,7 @@ public function execute(ViewExecutable $view) {
       }
 
       // Do sorting
-      if (!empty($this->orderby)) {
+      if ([] !== $this->orderby) {
```

Comment:
I don't get why this should be necessary, when `empty()` works perfectly fine, but why not ...

Reply:
I try to avoid `empty()` because it checks different things.